src/utils/hints/namesHints.js

```diff
@@ -0,0 +1,14 @@
+export const namesHints = {
+  name: 'Name that points to accounts, contracts, oracles, and NFTs.',
+  nameInAuction: 'Name that points to accounts, contracts, oracles, NFTs and more. Names shorter than 13 characters need to be auctioned.',
+  expires: 'Estimated date, time, and block height when the name will be detached from the address.',
+  buyer: 'Address of the account which paid for the Name and Name\'s price.',
```

Comment:
not sure if the part  " and Name\'s price" is needed

Reply:
removed